### DISCUSSION TITLE
WebRadio

### DISCUSSION THREAD

**Discussion by ploeffler** [arendst/Tasmota#16815] [Ideas]
some days ago I stumbled across [arduino-audio-tools ](https://github.com/pschatzmann/arduino-audio-tools) and thougth that having that included in Tasmota would make an easy web-radio, especially on ESP32

good or bad idea?

greets
peter

**Comment by sfromis** [comment]
If you imagine it in ESP8266, there is a probable issue of such a voluminous feature just not fitting alongside Tasmota. And there already exists other projects providing for an ESP8266 WebRadio to tinker with....
